### DISCUSSION TITLE
minimumReleaseAge for self-hosted instances

### DISCUSSION THREAD

**Discussion by 8L4ckc0FF33** [renovatebot/renovate#23409] [Suggest an Idea]
### Tell us more.

Hi there,

I really love working with renovate. Unfortunately I cannot use it at work because we have a really strict on-prem policy. Most of the features the self-hosted renovate instance offers are satisfying but for us it would only be feasible with the "minimumReleaseAge"-feature for the self-hosted instance. 
Would it be possible to also implement this feature for self-hosted instances?

Cheers,
blackcoffee!

**Comment by rarkins** [comment]
minimuReleaseAge is based on the release timestamp of each version, and it's supported everywhere including self-hosted

**Reply by 8L4ckc0FF33** [reply]
HI @rarkins, 

great to hear, thank you! But I am confused since it's not listed in the docs: https://docs.renovatebot.com/self-hosted-configuration/ Do they need to be updated?

**Reply by rarkins** [reply]
That just refers to config options which are ONLY for server side self hosted configuration
